synapse/pipeline.yml

```diff
@@ -267,17 +280,38 @@ steps:
             - /tmp/download-${BUILDKITE_BUILD_ID}/docker-compose.yaml
       - artifacts#v1.3.0:
           upload: [ "_trial_temp/*/*.log" ]
-#      - matrix-org/coveralls#v1.0:
-#          parallel: "true"
     retry: *retry_setup
 
   ################################################################################
   #
   # Sytest
   #
+  # Our tests have three dimensions:
+  #   1. Topology (Monolith, Workers, Workers w/ Redis)
+  #   2. Database (SQLite or PostgreSQL)
```

Comment:
also, we can run against *multiple* databases (`MULTI_POSTGRES`=1).

Reply:
Ah, good point. Our existing tests cover:

- Single Postgres (Monolith, Workers w/ Redis)
- Multi Postgres (Workers)

We do not cover:

- Single Postgres (Workers)
- Multi Postgres (Monolith, Workers w/ Redis)

I'll update the comment accordingly, but since this is the status quo I'll defer changing anything for a separate discussion and PR.

Reply:
Marking as resolved